cypress/integration/team/list.js

```diff
@@ -0,0 +1,26 @@
+describe('List team members', () => {
+  before(() => {
+      cy.task('db:seedTeamMembers')
+    }
+  )
+  it('Should not have permissions when not logged in', () => {
+    cy.visit('http://localhost:3000/team-members/list')
+
+    cy.contains('admin user').should('not.exist')
```

Comment:
This test is not completely correct, can you find out why?

Reply:
Nu stiu exact la ce te referi dar voi presupune ca la Welcome to Saleswell

Reply:
Nu-mi dau seama de ce este gresit. Daca nu avem acces la /team-members/list ne vom intarce la Wellcome to Saleswell si nu vom gasi niciun element de pe acea pagina.

Reply:
Yes, but the logic should be that you check the things you see, in which case you should also check that the "welcome" page has loaded.